project-base/storefront/tailwind.config.js

```diff
@@ -14,36 +14,28 @@ module.exports = {
             xxl: em(1560),
```

Comment:
Filter's visibility is broken 
<img width="223" alt="image" src="https://github.com/shopsys/shopsys/assets/55181354/ad089fbe-14e5-4432-b5e8-3ddfe818b677">